Realm/Tests/ObjectTests.m

```diff
@@ -785,7 +785,8 @@ - (void)testCreateInRealmValidationForDictionary {
         }
 
         if ([keyToInvalidate isEqualToString:@"anyCol"]) {
-            obj = self;
+            // Mixed properties now can accept dictionary or list, so we are skipping testing invalid values for this.
```

Comment:
This test is still applicable for Mixed properties and just needs to be supplying something which is not a valid value for them.